app/models/message.rb

```diff
@@ -0,0 +1,16 @@
+class Message < ApplicationRecord
+  belongs_to :sender, :foreign_key => :sender_id, class_name: 'User'
+  belongs_to :recipient, :foreign_key => :recipient_id, class_name: 'User'
+  
+  has_many :contents, dependent: :destroy
+  
+  validates_uniqueness_of :sender_id, :scope => :recipient_id
```

Comment:
This looks like it's ensuring that each recipient contains only unique sender ids, which would limit the number of messages between people to one. 

Please check me on this, but this doesn't seem quite right to me.

Reply:
I try to keep same two users have only one message.

Reply:
So users aren't allowed to send multiple messages back and forth between the same two users?

Reply:
Same two user can send multiple text(contents) each other, but they are keeping only one messaage. Message includes many contents between two same users. I am sorry. Name is confusing.